libs/SalesforceSDK/src/com/salesforce/androidsdk/rest/files/FileRequests.java

```diff
@@ -238,40 +250,4 @@ private static HttpEntity makeFileShare(String fileId, String entityId, String s
             throw new RuntimeException(e);
         }
     }
-
-    /**
-     * Build a request that can upload a new file to the server, this will
-     * create a new file at version 1.
-     * 
-     * @param theFile
-     *            The path of the local file to upload to the server.
-     * @param name
-     *            The name/title of this file.
-     * @param description
-     *            A description of the file.
-     * @param mimeType
-     *            The mime-type of the file, if known.
-     * @return A RestRequest that can perform this upload.
-     * 
-     * @throws UnsupportedEncodingException
-     */
-    public static RestRequest uploadFile(File theFile, String name, String description,
```

Comment:
I had to move this out of here, since there's no way to simply return an MPE request. It's all low level stuff using ```HttpUrlConnection```, so it didn't make sense to leave it here.